typescript/cli/src/utils/files.ts

```diff
@@ -163,6 +163,23 @@ export function prepNewArtifactsFiles(
   return newPaths;
 }
 
+/**
+ * Retrieves artifacts file metadata for the current command.
+ * @param dryRun whether or not the current command is being dry-run
+ * @returns the artifacts files
+ */
+export function getArtifactsFiles(
+  deploymentFile: ArtifactsFile,
+  configFile: ArtifactsFile,
+  dryRun: boolean,
+): Array<ArtifactsFile> {
```

Comment:
Not a big deal but here's what I'd suggest for this if it's going to be a generalized util:
```ts
export function getArtifactsFiles(defaultFiles: ArtifactsFile[], dryRun=false)
// OR
export function getArtifactsFiles(dryRun: boolean, ...defaultFiles: ArtifactsFile[])
```

Reply:
Really love this– refactored for the first form. Did not feel quite right first go but ended up moving on; very happy to make this change